os_net_config/schema.yaml

```diff
@@ -196,6 +198,8 @@ definitions:
                 anyOf:
                     - $ref: "#/definitions/string_or_param"
                     - $ref: "#/definitions/int_or_param"
+            metric:
+                $ref: "#/definitions/int_or_param"
```

Comment:
Both methods of specifying metrics is allowed now ? Currently we can do like this: https://github.com/os-net-config/os-net-config/blob/a1ec327a0654e435d5518e0b60b21d23434b45e6/etc/os-net-config/samples/routes.yaml#L17-L27

Reply:
That will lead to confusions if its specified at both places. Better to define in only one place.